ush/gefs_atmos_extractvars.sh

```diff
@@ -0,0 +1,102 @@
+#! /usr/bin/env bash
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+source "${USHgfs}/extractvars_tools.sh"
```

Comment:
If I move `source "${USHgfs}/extractvars_tools.sh"` from `gefs_atmos_extractvars.sh` to `exglobal_extractvars.sh`, the functions defined in `extractvars_tools.sh` cannot be found when they are executed in `exglobal_extractvars.sh` or any other ush script. This makes it necessary to source `preamble` and `extractvars_tools` in every script that they are used. It would be nice if I can source these just once in the exglobal script, and not in every ush script. Will revisit this to see if we can do this.